assimilation_code/modules/assimilation/filter_mod.f90

```diff
@@ -2022,10 +2022,10 @@ subroutine verbose_forward_op_output(qc_ens_handle, prior_post, ens_size, keys)
 endif
 
 ! qc_ens_handle is a real representing an integer; values /= 0 get written out
-do i = 1, ens_size
-   do j = 1, qc_ens_handle%my_num_vars
-      if(nint(qc_ens_handle%copies(i, j)) /= 0) write(forward_unit, *) i, keys(j), nint(qc_ens_handle%copies(i, j))
-   end do
+do i = 1, ens_size 
+    do j = 1, qc_ens_handle%my_num_vars 
+        if(nint(qc_ens_handle%copies(i, j)) /= 0) write(forward_unit, *) i, keys(qc_ens_handle%my_vars(j)), nint(qc_ens_handle%copies(i, j)) 
+    end do 
```

Comment:
This is a style comment:
 three spaces for indentation in dart, and theres some whitespace been added at the end of lines. 

It is good to separate style changes (e.g. whitespace) from code changes.  This is a small pull request so it is easy to see what is going on, but you can imagine in a larger pull request if there are a lot of style changes it can be harder to review any code that has changed.